package.json

```
@@ -1,42 +1,43 @@
 {
   "name": "prettier-tslint",
-  "description":
-    "Formats your TypeScript using Prettier followed by tslint --fix",
+  "description": "Formats your TypeScript using Prettier followed by tslint --fix",
   "version": "0.0.0-development",
   "author": "Lucas Azzola <@azz>",
   "license": "MIT",
   "main": "dist",
   "bin": "./bin/prettier-tslint.js",
-  "files": ["bin", "dist"],
+  "files": [
+    "bin",
+    "dist"
+  ],
   "dependencies": {
-    "chalk": "^2.4.0",
+    "chalk": "^2.4.1",
     "globby": "^8.0.1",
-    "ignore": "^3.3.7",
+    "ignore": "^5.0.2",
     "require-relative": "^0.8.7",
-    "tslint": "^5.9.1",
-    "yargs": "^11.0.0"
+    "tslint": "^5.11.0",
+    "yargs": "^12.0.2"
   },
   "peerDependencies": {
-    "prettier": "^1.7.4",
-    "typescript": "^2.5.3"
+    "prettier": "^1.14.3",
+    "typescript": "^3.1.3"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "eslint": "^4.19.1",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-prettier": "^2.6.0",
-    "jest": "^21.2.1",
-    "prettier": "1.12.1",
-    "semantic-release": "^8.0.3",
-    "typescript": "~2.8.3"
+    "babel-preset-env": "^1.7.0",
+    "eslint": "^5.7.0",
+    "eslint-config-prettier": "^3.1.0",
+    "eslint-plugin-prettier": "^3.0.0",
+    "jest": "^23.6.0",
+    "prettier": "^1.14.3",
+    "semantic-release": "^15.10.3",
```

```suggestion
    "semantic-release": "^8.0.3",
```

I'd like to keep this the same as there are a lot of breaking changes, would also need to upgrade the config.